usagov/vote-gov-drupal
# Feature/vote 1879 rework deploy pipeline

## Jira ticket

[Vote-1879](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-1879&assignee=611272e0eb7aef00694b8a97)

## Description

This PR will update the pipeline for the new flow of deploying and will also remove the deploy-test job.  Also sets up to add cypress testing for the front end static site 

## Deployment and testing

### Pre-deploy

1. n/a

### Post-deploy

1. Need to set up PR requirements for the tests to allow merge -> php-lint and snyk need to pass on all branches to merge.  Cypress can fail on Dev and Test but need to pass for Dev and Main

### QA/Test

1. Review file change and make sure that deploy-test has been removed and jobs reflect the list of steps in the ticket. 

## Checklist for the Developer




## Checklist for the Peer Reviewers
